packages/google_maps_flutter/google_maps_flutter/pubspec.yaml

```diff
@@ -2,7 +2,7 @@ name: google_maps_flutter
 description: A Flutter plugin for integrating Google Maps in iOS and Android applications.
 repository: https://github.com/flutter/packages/tree/main/packages/google_maps_flutter/google_maps_flutter
 issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3A%22p%3A+maps%22
-version: 2.4.0
+version: 2.4.1
```

Comment:
2.5.0, since it's a new feature.

Reply:
Version updated to 2.5.0 https://github.com/flutter/packages/pull/3682/commits/761d06c88cbc0710742ecf9d92d78f395bb8faec